### ISSUE TITLE
Error when bulk inserting into a table for an entity with ignored properties

### ISSUE THREAD

**Issue by jpett** [borisdj/EFCore.BulkExtensions#89]
I have an entity with certain properties in an owned type that are not mapped to the database (by specifying builder.Ignore() in fluent API). Trying to bulk insert into the entity's table I receive the error

`Input array is longer than the number of columns in this table.`

I think I found the problem in SqlBulkOperation.GetDataTable: When creating the DataTable the EF mapping information is used to obtain the list of ownedEntityProperties:

`var ownedEntityProperties = ownedEntityType.GetProperties().ToList();`

ownedEnttiyType is of type Microsoft.EntityFrameworkCore.Metadata.IEntityType. Accordingly, ignored properties are not included in ownedEntityProperties.

When filling the actual data into the DataTable created, however, the PropertyInfo array obtained via System.Reflections is used. Thus, the properties ignored in the mapping are added as well which results in the error described above.

I think I was able to fix this problem and will try to create a pull request soon.

**Comment by borisdj** [comment]
I have accepted your pull request but then changed(simplified) the fix.
Could you test if new NuGet 2.2.0 works for your case.
I have tested it with this structure and it passes:
```C#
public class ChangeLog
{
    public int ChangeLogId { get; set; }

    public string Description { get; set; }

    public Audit Audit { get; set; }
}

[Owned]
public class Audit
{
    public string ChangedBy { get; set; }

    [NotMapped]
    public DateTime? ChangedTime { get; set; }
}
```

**Comment by jpett** [comment]
Thank you very much for merging my fix! Unfortunately, with NuGet Package 2.2.2 I still get an error in my program:

 System.ArgumentException: Input array is longer than the number of columns in this table.
   at System.Data.DataTable.NewRecordFromArray(Object[] value)
   at System.Data.DataRowCollection.Add(Object[] values)
   at EFCore.BulkExtensions.SqlBulkOperation.GetDataTable[T](DbContext context, IList`1 entities, SqlBulkCopy sqlBulkCopy)
   at EFCore.BulkExtensions.SqlBulkOperation.Insert[T](DbContext context, IList`1 entities, TableInfo tableInfo, Action`1 progress)

The same program works fine with the fork I created for my pull request.

**Comment by jpett** [comment]
Sorry, I deleted my previous comment which stated that the problem persists because I thought I had made a mistake when testing the new release. But after a second test, I can confirm the issue is still there. Unfortunately, my model is quite complex and not suited for a short example. 

One difference to your example, however, is that I have ignored properties in the parent class and in the owned types. So maybe adding a [NotMapped] Property to ChangeLog might reproduce the problem.
